cmd/kube-apiserver/app/options/options_test.go

```diff
@@ -130,6 +130,9 @@ func TestAddFlags(t *testing.T) {
 		"--emulated-version=test=1.31",
 		"--emulation-forward-compatible=true",
 		"--runtime-config-emulation-forward-compatible=true",
+		"--cle-lease-duration=10s",
+		"--cle-renew-deadline=5s",
+		"--cle-retry-period=1s",
```

Comment:
nit: We typically avoid using acronyms in flag names.  Can we find some prefix that's not horribly verbose as an alternative to "cle" here?